specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -28,12 +28,10 @@ These are the global settings for the KubernetesConfiguration.
 title: SourceControlConfigurationClient
 description: KubernetesConfiguration Client
 openapi-type: arm
-tag: package-2022-03
+tag: package-preview-2022-01-15
```

Comment:
Let us do not downgrade the tag. Could you please revert this change?
Our SDK releasing process is depending on this tag, downgrade this will make the SDK to downgrade by api-versions

Reply:
Ok I will revert this change. 

Can you please let me know what I need to change regarding this comment you made previously then: "Please also change the default tag to the new tag added in this PR."
I am not sure what the tag needs to be changed to then.